demisto_sdk/commands/content_graph/strict_objects/pack_meta_data.py

```diff
@@ -0,0 +1,65 @@
+from typing import List, Optional, Union
+
+from pydantic import Field
+
+from demisto_sdk.commands.common.constants import MarketplaceVersions
+from demisto_sdk.commands.content_graph.strict_objects.common import BaseStrictModel
+
+
+class StrictPackMetadata(BaseStrictModel):
+    name: str
+    display_name: Optional[str] = None
+    description: Optional[str] = None
+    created: Optional[str] = None
+    updated: Optional[str] = None
+    legacy: Optional[bool] = None
+    support: str
```

Comment:
must be `xsoar`, `partner`, `community` or `developer`. If we don't have an Enum for this somewhere, it's a good opportunity to create one under `constants.py`

Reply:
already exists, I'll use it